darts/models/forecasting/theta.py

```diff
@@ -182,6 +182,9 @@ def min_train_series_length(self) -> int:
         else:
             return 3
 
+    def _is_probabilistic(self) -> bool:
```

Comment:
this one can be removed as well :) after that all looks good!

Reply:
;) missed this one, thanks